utils/nctl/sh/assets/setup.sh

```diff
@@ -63,14 +63,14 @@ function _set_chainspec() {
     GENESIS_TIMESTAMP=$(get_genesis_timestamp $3)
     sed -i "s/^\([[:alnum:]_]*timestamp\) = .*/\1 = \"${GENESIS_TIMESTAMP}\"/" $path_config > /dev/null 2>&1
 
-    # Set config settings:
+    # Override config settings as all paths need to point relative to nctl's assets dir:
+    #    genesis.accounts_path
     #    genesis.mint_installer_path
     #    genesis.pos_installer_path
     #    genesis.standard_payment_installer_path
     #    genesis.auction_installer_path
-    # These are paths to WASM smart contracts, and they still need to point
-    # relative to nctl's assets dir.
-    sed -i "s?\.\./\.\./target/wasm32-unknown-unknown/release/?../bin/?g" $path_config > /dev/null 2>&1
+    sed -i "s?\${BASEDIR}/target/wasm32-unknown-unknown/release/?../bin/?g" $path_config > /dev/null 2>&1
```

Comment:
This might need to have the backslash before `${BASEDIR}` removed?

Reply:
No is good ... tested it a few times.